$NetBSD: patch-ab,v 1.3 2007/01/14 15:19:04 rillig Exp $

This patch supports the pkgsrc setting BDB185_DEFAULT=db4. According to
its author, David A. Holland, it is still a hack.

From PR 35417.

--- libcob/fileio.c.orig	2005-05-03 05:31:14.000000000 -0400
+++ libcob/fileio.c	2007-01-13 11:42:01.000000000 -0500
@@ -48,7 +48,11 @@
 #endif
 
 #if HAVE_DBOPEN
+#if defined(WITH_DB)
+#include <db_185.h>
+#else
 #include <db.h>
+#endif
 #else
 #if HAVE_DB1_DB_H
 #include <db1/db.h>
